requirements-app.txt

```diff
@@ -10,7 +10,7 @@ SQLAlchemy==1.1.4
 SQLAlchemy-Utils==0.30.5
 
 git+https://github.com/alphagov/digitalmarketplace-utils.git@29.0.0#egg=digitalmarketplace-utils==29.0.0
-git+https://github.com/alphagov/digitalmarketplace-apiclient.git@10.8.0#egg=digitalmarketplace-apiclient==10.8.0
+git+https://github.com/alphagov/digitalmarketplace-apiclient.git@11.4.0#egg=digitalmarketplace-apiclient==11.4.0
```

Comment:
Is there a breaking change here?

Reply:
Yes but it doesn't affect API (which just needs the new audit event type).